plugins/woocommerce/client/admin/config/core.json

```
@@ -36,6 +36,7 @@
 		"woo-mobile-welcome": true,
 		"wc-pay-promotion": true,
 		"wc-pay-welcome-page": true,
-		"async-product-editor-category-field": false
+		"async-product-editor-category-field": false,
+		"launch_your_store": false
```

nit: I think it would be great that we can stick with the same naming convention as above.

```suggestion
		"launch-your-store": false
```